app/lib/features/chat/widgets/bubble_builder.dart

```diff
@@ -21,7 +21,9 @@ import 'package:swipe_to/swipe_to.dart';
 
 final _log = Logger('a3::chat::bubble_builder');
 
-class BubbleBuilder extends ConsumerWidget {
+// keep this widget stateful consumer
+// otherwise doesn't hold correct message state
+class BubbleBuilder extends ConsumerStatefulWidget {
```

Comment:
but ... does that mean our internal provider _doesn't_ have the correct state? Does that mean that once I leave the room and re-enter it, it will show the wrong state? Why would we make the _widget_ be the single source of truth about state if the we used _riverpod_ for state management?

Reply:
but the question is why? Why would that come about?